examples/load-ampcontext-creative-runtime-not-at-top.html

```diff
@@ -0,0 +1,6 @@
+<html>
```

Comment:
As i see such a requirement more and more often (to test a page inside an iframe).
I suggest we introduce a new endpoint in our `server.js`.

it should handle request like `/iframe/examples/loads-windowcontext-creative.max.html`, and dynamically return this HTML.

Let me know if you don't know how to do it, I can help.